### DISCUSSION TITLE
How to update subscription properties without unsubscribing subscription?

### DISCUSSION THREAD

**Discussion by lishengye** [apache/pulsar#20740] [Q&A] [answered]
Hi, i'm trying use the entry filter to filter message by tag, However it seems subscription properties is fixed when subscription is established, i don't konw how to update it dynamically

**Comment by tisonkun** [comment]
cc @Technoboy- @BewareMyPower do you have some inputs here? Or is it impossible in Pulsar design?

**Reply by BewareMyPower** [reply, marked as answer]
It's impossible. I think the entry filter should not be used to modify the internal status like the subscription properties.
